pkg/netceptor/netceptor.go

```diff
@@ -587,43 +591,48 @@ func fixedLenBytesFromString(s string, l int) []byte {
 
 // Translates an incoming message from wire protocol to messageData object.
 func (s *Netceptor) translateDataToMessage(data []byte) (*messageData, error) {
-	if len(data) < 33 {
+	if len(data) < 36 {
```

Comment:
should we make a `MinimumPacketSize` const for this

Reply:
The two functions `translateDataToMessage` and `translateDataFromMessage` are the only places where these values are used.  I think the functions are considerably more readable with the values inline like this, rather than defined in constants far away from the code.  If the values needed to be used anywhere else, I would agree with you, but as things stand I think it's better to leave this as is.